### ISSUE TITLE
simplify the unit tests

### ISSUE THREAD

**Issue by yaroslavya** [kukeiko/type-vein#2]
I suggest a simpler unit tests to be added. Their main goal is to be very easy to understand so anyone can understand the basics by just looking at them. The more complex scenarios are of course covered by unit tests, but after understanding the easy ones the complex ones should be more straightforward.

**Comment by kukeiko** [comment]
I think we should start by taking your getting-started.spec from #1  (autoformat it as well 😀), maybe wrap the describe in an additional layer?

But in the pull request you mentioned to not pull, it's just for comms?

**Comment by yaroslavya** [comment]
Yep, its just to look into it, discuss and make a better PR that is to be merged.

**Comment by kukeiko** [comment]
I wrote the first system test and added comments to illustrate what's going on. The test showcases a simple example of how you load data via type-vein: https://github.com/kukeiko/type-vein/blob/master/test/system/loading-data.spec.ts
